scripts/unix_share_build.sh

```diff
@@ -271,9 +272,9 @@ function linux_build_plugin_pkg(){
     cd ..
     ln -sv svalinn/libsvalinn_plugin.so .
     cd ../..
-    tar --sort=name -czvf svalinn-plugin_${OS}_cubit_$1.tgz bin
-    chmod 666 svalinn-plugin_${OS}_cubit_$1.tgz
-    cp svalinn-plugin_${OS}_cubit_$1.tgz $SCRIPTPATH/release/
+    tar --sort=name -czvf svalinn-plugin_${OS}-${OS_VERSION}_cubit_$1.tgz bin
```

Comment:
It probably makes sense to assign this tarball file name to a variable and reuse it over these three lines to prevent typos causing errors